### PR TITLE
[Oreo] Downloadable Fonts Sample

### PR DESCRIPTION
**Pull request by gonzalonm**
Ported sample from https://github.com/googlesamples/android-DownloadableFonts/

### CODE REVIEW

**Comment by pjcollins** [comment]
@gonzalonm could we rebase this to exclude the unrelated markdown fix-ups?

File: android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	{
+		String FamilyName = null;
+
+		Float Width = null;
+
+		Integer Weight = null;
+
+		Float Italic = null;
+
+		Java.Lang.Boolean Besteffort = null;
```
Comment (pjcollins) [inline android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs]: Can use a bool (System.Boolean) here and avoid the need to construct a new Java.Lang.Boolean instance later.

Comment (gonzalonm) [reply android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs]: @pjcollins Yes, but I don't know how to check if that property was assigned or not. Required here: https://github.com/xamarin/monodroid-samples/pull/203/files/db1794b67fa36cefda92307ee31cce11dc2123ac#diff-adade3c305f996bafb2c367743eb5490R85

Comment (pjcollins) [reply android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs]: Ahh, using `bool?` will allow it to be nullable. 
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/using-nullable-types

Comment (gonzalonm) [reply android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs]: Nice! I thought it from Java perspective 👍

**Comment by gonzalonm** [comment]
@pjcollins Applied feedback

**Pull request merged**